spec/lib/disability_compensation/providers/lighthouse_rated_disabilities_provider_spec.rb

```diff
@@ -7,11 +7,5 @@
 RSpec.describe LighthouseRatedDisabilitiesProvider do
   let(:current_user) { build(:user, :loa3) }
 
-  # TODO: update when the LighthouseRatedDisabilitiesProvider is implemented
-  it 'behaves like a RatedDisabilitiesProvider' do
-    # it_behaves_like :rated_disabilities_provider
-    expect do
-      LighthouseRatedDisabilitiesProvider.new(current_user)
-    end.to raise_error NotImplementedError
-  end
+  it_behaves_like 'rated disabilities provider'
```

Comment:
Is `it_behaves_like` effective on its own? If I remember, it shares examples with the current context, but then it requires `shared_examples` and the relevant test statements

Reply:
I'm blind, I just noticed the includes, though perhaps the `shared_examples` could be made more robust and non-specific for even more reuse